addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -833,18 +833,22 @@ export class OdooEditor extends EventTarget {
      * Place the selection on the last known selection position from the history
      * steps.
      *
+     * @param {boolean} [inEditable=false]
```

Comment:
I had to look at the code to understand what this did. `limitToEditable` or `confineToEditable` maybe? And/or some mention of it in the docstring?